dev/com.ibm.websphere.security.wim.base/resources/com/ibm/websphere/security/wim/util/resources/WebsphereWimUtilMessages.nlsprops

```diff
@@ -18,6 +18,15 @@
 
 # Message prefix block: Profile Manager: 1000-1999
 
+
+DUPLICATE_PROPERTY_EXTENDED=CWIML1996W: The extended property ''{0}'' has already been defined for the ''{1}'' entity and will be ignored.
```

Comment:
DUPLICATE_PROPERTY_EXTENDED=CWIML1996W: The extended property ''{0}'' has already been defined for the ''{1}'' entity, and will be ignored.

**Question, is "{0}" ignored, or is "{1}" ignored? The way it's phrased, it's not clear. Is this correct?:**

DUPLICATE_PROPERTY_EXTENDED=CWIML1996W: The extended property ''{0}'' is ignored because it was defined for the ''{1}'' entity.

Reply:
Hi @jvanhill, great. That doesn't change my suggestions. Thanks!